frontend/widgets/OBSBasic.hpp

```diff
@@ -1470,6 +1470,14 @@ private slots:
 	std::vector<OBSDataAutoRelease> safeModeTransitions;
 	QPointer<QPushButton> transitionButton;
 	QPointer<QMenu> perSceneTransitionMenu;
+
+	std::unordered_map<std::string, OBSSource> transitions;
+	/* NOTE: There is a reliance on the order of insertion */
+	std::vector<std::string> transitionUuids;
+	/* FIXME: Replace usages of a name to identify a transition */
```

Comment:
Is this a `FIXME` for future changes or for this PR? (Should also use C++ comment style).

Reply:
In the long term we want to rely manly on UUIDs, some part of the code still have reliance on the name of the transition.
In this PR, I tried to minimized it.